### ISSUE TITLE
You forgot color in SlideAction

### ISSUE THREAD

**Issue by boygiandi** [letsar/flutter_slidable#124]
In SlideAction class, when you call super(), please add color parameter.

```
SlideAction({
    Key key,
    @required this.child,
    VoidCallback onTap,
    Color color,
    Decoration decoration,
    bool closeOnTap = _kCloseOnTap,
  })  : assert(child != null),
        assert(decoration == null || decoration.debugAssertIsValid()),
        assert(
            color == null || decoration == null,
            'Cannot provide both a color and a decoration\n'
            'The color argument is just a shorthand for "decoration:  BoxDecoration(color: color)".'),
        decoration =
            decoration ?? (color != null ? BoxDecoration(color: color) : null),
        super(
          key: key,
          onTap: onTap,
          closeOnTap: closeOnTap,
          **color: color**
        );
```

**Comment by BizzottoClaudio** [comment]
this would be useful

**Comment by letsar** [comment]
Fixed in 0.5.7
